src/main/java/com/zepben/cimbend/get/NetworkConsumerClient.kt

```diff
@@ -96,60 +103,117 @@ class NetworkConsumerClient(
         }
     }
 
+    /***
+     * Retrieve the feeder network for the specified [mRID] and store the results in the [service].
+     *
+     * This is a convenience method that will fetch the feeder object, all of the equipment referenced by the feeder (normal state),
+     * the terminals of all elements, the connectivity between terminals, the locations of all elements, the ends of all transformers
+     * and the wire info for all conductors.
+     *
+     * @return The [Feeder], or null if it was nto found.
+     */
+    fun getFeeder(service: NetworkService, mRID: String): GrpcResult<Feeder> {
+        return tryRpc {
```

Comment:
I was a bit confused when I was reading this function. I saw the `throw feederResponse.thrown!!` which made me think "why is this throwing, shouldn't it be returning a new GrpcResult with the thrown (and !! always raises a red flag for me)?"  Then I realised this whole thing was warpped in a tryRpc (so nested tryRpc) and it would be catching. 

Personally I think it would be cleaner to add something like a `onError` (and prob a corresponding `onSuccess` to the GrpcResult, so you can then do something like this:
```kotlin
val feeder = getIdentifiedObject(service, mRID)
    .onError { return@getFeeder GrpcResult.ofError(it) }
    .result
```
and
```kotlin
getIdentifiedObjects(service, service.getUnresolvedReferenceMrids(Resolvers.equipment(feeder)))
    .onError { return@getFeeder GrpcResult.ofError(it) }
```

This would remove the need to use tryRpc in this function where no RPC calls are directly made and also remove the need for the validateRpcCall.

What do you think?